### ISSUE TITLE
Keep x and add ɣ

### ISSUE THREAD

**Issue by trans** [LogosInitiative/logla#8]
If `x` is to remain then it's voiced counterpart `ɣ` should be added as well.

**Comment by thetommoore** [comment]
I agree, though this raises an orthography issue; perhaps /ɣ/ is ⟨q⟩?

**Comment by trans** [comment]
Yea, ⟨q⟩ could work.

I thought an alternative might be to deprecate /x/ and only allow these two phones as digraphs using /k/ and /g/. But in combination with what?

**Comment by thetommoore** [comment]
Why not h? It's not used elsewhere.
